### ISSUE TITLE
Cmake ENABLE_DOCS=ON needs cuda

### ISSUE THREAD

**Issue by PhilipDeegan** [LLNL/SAMRAI#255]
somewhere along the lines our build just for samrai documentation, has started to fail with the error 

```
/usr/bin/ld: cannot find -lcuda
```

cmake config

```
cmake -D CMAKE_C_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
      -DCMAKE_INSTALL_PREFIX=/root/.local-samrai -DENABLE_DOCS=ON \
      -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-O2" ..
```

cmake config output 

```
 -- BLT MPI Compile Flags:  -pthread
 -- BLT MPI Include Paths:  /usr/include/openmpi-x86_64;/usr/lib64/openmpi/lib
 -- BLT MPI Libraries:      /usr/lib64/openmpi/lib/libmpi.so;/usr/lib64/openmpi/lib/libmpi_cxx.so;/usr/lib64/openmpi/lib/libmpi_usempif08.so;/usr/lib64/openmpi/lib/libmpi_usempi_ignore_tkr.so;/usr/lib64/openmpi/lib/libmpi_mpifh.so
 -- BLT MPI Link Flags:     SHELL:-Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -pthread
 -- MPI Executable:       /usr/lib64/openmpi/bin/mpiexec
 -- MPI Num Proc Flag:    -n
 -- MPI Command Append:
 -- Using MPI Fortran header: mpif.h
 -- OpenMP Support is Off
 -- CUDA Support is Off
```

failing command

```
[ 58%] Linking CXX executable ../../../bin/sparse

...

/usr/bin/c++  -Wall -Wextra  -O2    -g -rdynamic -Wl,-rpath -Wl,/usr/lib64/openmpi/lib -Wl,--enable-new-dtags -pthread CMakeFiles/mblktree.dir/main-mbtree.cpp.o -o ../../../bin/mblktree  -Wl,-rpath,/usr/lib64/openmpi/lib ../../../lib/libSAMRAI_hier.a ../../../lib/libSAMRAI_geom.a ../../../lib/libSAMRAI_mesh.a ../../../lib/libSAMRAI_tbox.a -lcuda ../../../lib/libSAMRAI_pdat.a ../../../lib/libSAMRAI_xfer.a ../../../lib/libSAMRAI_hier.a ../../../lib/libSAMRAI_tbox.a /usr/lib64/openmpi/lib/libmpi.so /usr/lib64/openmpi/lib/libmpi_cxx.so /usr/lib64/openmpi/lib/libmpi_usempif08.so /usr/lib64/openmpi/lib/libmpi_usempi_ignore_tkr.so /usr/lib64/openmpi/lib/libmpi_mpifh.so /usr/lib64/openmpi/lib/libhdf5.so -lgfortran -lquadmath
```

**Comment by PhilipDeegan** [comment]
issue appears to manifest March 3rd 2023 with `Version 4.2.0` release

**Comment by PhilipDeegan** [comment]
closed by #262
